test/test-remote/test_loki_log_ingest_api.ts

```diff
@@ -150,7 +151,7 @@ suite("Loki API test suite", function () {
   });
 
   test("insert w/ cntnrzd FluentD(loki plugin), then query", async function () {
-    const sampleTimeRFC3339nano = "2010-10-10T10:10:01.123456789Z";
+    const now = timestampToRFC3339Nano(ZonedDateTime.now(ZoneId.UTC));
```

Comment:
update: see discussion above. This change makes sense because we do not want to test inserting data older than the retention period.